tests/Doctrine/Tests/Common/DataFixtures/ReferenceRepositoryTest.php

```diff
@@ -129,15 +128,21 @@ public function testUndefinedReference(): void
 
     public function testThrowsExceptionAddingDuplicatedReference(): void
     {
-        $referenceRepository = new ReferenceRepository($this->getMockSqliteEntityManager());
-        $referenceRepository->addReference('duplicated_reference', new stdClass());
+        $em                  = $this->getMockSqliteEntityManager();
+        $referenceRepository = new ReferenceRepository($em);
+
+        $schemaTool = new SchemaTool($em);
+        $schemaTool->dropSchema([]);
+        $schemaTool->createSchema([$em->getClassMetadata(Role::class)]);
```

Comment:
Are these lines necessary? (since apparently there are no calls to `persist` or `flush`)

Reply:
No need to make a new commit, please amend the first one.